packages/react/src/analytics/integrations/GA4/validation/eventSchemas.js

```diff
@@ -188,10 +188,10 @@ const changeSizeProductInCartSchema = updateProductInCart.concat(
     oldSize: yup.string(),
     size: yup
       .string()
-      .test(errorCodes.InvalidOldSize, errorCodes.InvalidOldSize, function () {
-        const { oldSize } = this.parent;
+      .test(errorCodes.InvalidSize, errorCodes.InvalidSize, function () {
```

Comment:
Question: Will this condition ever happen? I believe `getProductUpdatedEventList` will only send the internal change size event when the size have changed. I believe these validations (not only for size, but quantity and colour as well) will need to go for the `PRODUCT_UPDATED` event.